tests/Unit/ProtoTests/Basic/out/src/Client/BasicClient.php

```diff
@@ -27,10 +27,12 @@
 use Google\ApiCore\ApiException;
 use Google\ApiCore\CredentialsWrapper;
 use Google\ApiCore\GapicClientTrait;
+use Google\ApiCore\InsecureCredentialsWrapper;
```

Comment:
We need to remember to update these clients to have a minimum version of `google/gax:^1.29` before we roll any of these out. Otherwise, this will throw a class not found error in versions of GAX < 1.29.

I think the best way to handle this may be to just update ALL minimum versions of GAX to 1.29 now - or at the very least, update `Spanner/composer.json`.

Reply:
Updated the min gax version to 1.29.1. PR: https://github.com/googleapis/google-cloud-php/pull/7095